docs: add quickstart dockerfile with default sqlite database (#7952)
langflow-ai/langflow: docs/docs/Deployment/deployment-docker.md

This guide demonstrates deploying Langflow with Docker and Docker Compose.

Three options are available:

> * The [Quickstart](#quickstart-with-a-sqlite-database) option starts a Docker container with default values.

Author: ```suggestion
* The [Quickstart](#quickstart) option starts a Docker container with default values.
```
```suggestion
* The [Quickstart](#quickstart) option starts a Docker container with default values.
```

> * The [Quickstart](#quickstart-with-a-sqlite-database) option builds a Langflow image with Langflow's default SQLite database.

Author: ```suggestion
* The [Quickstart](#quickstart-with-a-sqlite-database) option builds a Langflow image with Langflow's default SQLite database.Create a Dockerfile with Langflow's default SQLite database. The SQLite database is stored in the container's filesystem, which means all data is lost when the container is stopped since no persistent volume is configured.
```

Reviewer: I suggest we don't include SQLite as it's unnecessary details.

> * The [Quickstart](#quickstart-with-a-sqlite-database) option starts a Docker container with default values. Langflow's default SQLite database is stored in the container's filesystem, which means all data is lost when the container is stopped since no persistent volume is configured.

Author: ```suggestion
* The [Quickstart](#quickstart-with-a-sqlite-database) option starts a Docker container with default values.
```

This guide demonstrates deploying Langflow with Docker and Docker Compose.

Three options are available:

* The [Quickstart](#quickstart-with-a-sqlite-database) option builds a Langflow image with Langflow's default SQLite database.
* The [Docker compose](#clone-the-repo-and-build-the-langflow-docker-container) option builds Langflow with a persistent PostgreSQL database service.

> * The [Package your flow as a docker image](#package-your-flow-as-a-docker-image) option demonstrates packaging an existing flow with a Dockerfile.

Reviewer 2: ```suggestion
* The [Package your flow as a docker image](#package-your-flow-as-a-Docker-image) option demonstrates packaging an existing flow with a Dockerfile.
```

For more information on configuring the Docker image, see [Customize the Langflow Docker image with your own code](#customize-the-langflow-docker-image-with-your-own-code).

## Prerequisites

- [Docker](https://docs.docker.com/)
- [Docker Compose](https://docs.docker.com/compose/)

> ## Quickstart with a SQLite database
> 
> Create a Dockerfile with Langflow's default SQLite database.
> The SQLite database is stored in the container's filesystem, which means all data is lost when the container is stopped since no persistent volume is configured.
> 
> To use the Dockerfile:
> 
> 1. Create a new directory for your project:
>    ```bash
>    mkdir langflow-docker && cd langflow-docker
>    ```
> 
> 2. Create the Dockerfile with the following configuration.
> 
> ```dockerfile
> # Use the official Langflow image
> FROM langflowai/langflow:latest
> 
> # Expose the default port
> EXPOSE 7860
> 
> # Run Langflow
> CMD ["python", "-m", "langflow", "run", "--host", "0.0.0.0", "--port", "7860"]
> ```
> 
> 3. Build and run the container:
>    ```bash
>    docker build -t langflow-docker .
>    docker run -p 7860:7860 langflow-docker
>    ```
> 
> Langflow is accessible at `http://localhost:7860/`.
> 

Reviewer: I feel like this is not really a _quick_ start. Consider instead

```suggestion
## Quickstart

With Docker installed and running on your system, run this command:

`docker run -p 7860:7860 langflowai/langflow`

Langflow is now accessible at `http://localhost:7860/`.
```
